example_scripts/galaxy_zoo_touch_table/prepare_db_from_classification_export.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+import pandas as pd
+import json
+import string
+from tqdm import tqdm
```

Comment:
I'd probably do `from tqdm import tqdm as progress_bar`, so folks unfamiliar with either this library or the Arabic language will understand what it is :P